lib/irc/ClientPool.js

```diff
@@ -238,25 +238,31 @@ ClientPool.prototype._onClientDisconnected = function(bridgedClient) {
     // the timeout below holds it in a closure, preventing it from being GC'd.
     bridgedClient = undefined;
     setTimeout(function() {
-        cli.connect().then(function() {
-            log.info(
-                "<%s> Reconnected %s@%s", cli._id, cli.nick, cli.server.domain
-            );
-            if (chanList.length > 0) {
-                log.info("<%s> Rejoining %s channels", cli._id, chanList.length);
-                chanList.forEach(function(c) {
-                    cli.joinChannel(c);
-                });
-            }
-            self._sendConnectionMetric(cli.server);
-        }, function(e) {
-            log.error(
-                "<%s> Failed to reconnect %s@%s", cli._id, cli.nick, cli.server.domain
-            );
-        });
+        self._reconnect(cli, chanList);
```

Comment:
I know this is leftover from when we previously did reconnects from the `ClientPool`, but this just adds needlessly to the diff. Can you revert this file (but keep the change to `RECONNECT_TIME_MS`)?